src/ophyd_async/planstubs/ensure_connected.py

```diff
@@ -0,0 +1,22 @@
+import bluesky.plan_stubs as bps
+
+from ophyd_async.core.device import Device
+from ophyd_async.core.utils import DEFAULT_TIMEOUT, wait_for_connection
+
+
+def ensure_connected(
+    *devices: Device,
+    sim: bool = False,
+    timeout: float = DEFAULT_TIMEOUT,
+    force_reconnect=False,
+):
+    yield from bps.wait_for(
+        [
+            lambda: wait_for_connection(
```

Comment:
Why does this have to be a lambda?

Reply:
Discussed here:

https://blueskyproject.io/bluesky/main/msg.html#wait-for

The `wait_for` planstub is weird.